frontend/packages/integration-tests-cypress/tests/crud/secrets/key-value.cy.ts

```diff
@@ -134,4 +139,13 @@ describe('Create key/value secrets', () => {
       });
     });
   });
+
+  it('Validate Base64 data value for java keystore secret not changing', () => {
+    // Add for customer Bug 2070562 Base64 data value for java keystore secret changing automatically...
+    cy.exec(`oc get secret ${jksSecretName} -n ${testName} -o yaml > /tmp/secret1.yaml`);
+    cy.visit(`/k8s/ns/${testName}/secrets/${jksSecretName}/edit`);
+    secrets.save();
+    cy.exec(`oc get secret -n ${testName} ${jksSecretName} -o yaml > /tmp/secret2.yaml`);
+    cy.exec(`diff /tmp/secret1.yaml /tmp/secret2.yaml`).its('stdout').should('eq', '');
+  });
```

Comment:
it looks like this is a duplicate scenario with `Validate create and edit of a key/value secret whose value is a binary file`?

Reply:
JKS file is a special type of binary file. This case was added to address a customer-reported bug, so it remains here intentionally, but I can remove it if you think it’s not necessary